.github/workflows/cli-test.yml

```diff
@@ -0,0 +1,42 @@
+name: CLI / Tests
+
+on:
+  push:
+    paths:
+      - 'packages/shortest/**'
+      - '.github/workflows/cli-*.yml'
```

Comment:
Grouped CLI-related jobs by `cli-*` prefix to self-document their context, and to trigger checks conditionally (e.g. trigger CLI-related checks when CLI code changes)